### DISCUSSION TITLE
Get plotter from the pyvista mesh display method

### DISCUSSION THREAD

**Discussion by zouki-dev** [ansys/pyfluent#1750] [Q&A]
Hello, first, note that I did create a post about this issue here : https://github.com/ansys/pyfluent-visualization/discussions/290
But it doesn't look like there are a lot of activity in this pyfluent-visualization channel.

**********************************************************************************************************************

I am looking to recover the plotter used to make the pyvista figure. Also I need the figure to be hidden. I am going to show the plotter in another window (using streamlit).

Here is what I am trying to use:

```
graphics = Graphics(session=solver)
mesh1 = graphics.Meshes["outlet-1"]
mesh1.show_edges = True
a = mesh1.surfaces_list.allowed_values
mesh1.surfaces_list = a
mesh1.display("window-0")
```

So I want to recover the plotter of pyvista and I want a hide_window parameter to pass to pyvista.

**Comment by prmukherj** [comment]
from ansys.fluent.visualization.pyvista import pyvista_windows_manager

p=pyvista_windows_manager.get_plotter("window-1")
p.view_xy()

I think @ajain-work can help if any further queries are there. 

Additional info: https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Renderer.view_isometric.html
